taxi/views.py

```diff
@@ -28,6 +30,17 @@ def index(request):
     return render(request, "taxi/index.html", context=context)
 
 
+@login_required
+def change_driver(request: HttpRequest, pk: int) -> HttpResponse:
+    car = Car.objects.get(id=pk)
+    driver = Driver.objects.get(id=request.user.id)
+    if driver.cars.contains(car):
+        driver.cars.remove(car)
+    else:
+        driver.cars.add(car)
+    return render(request, "taxi/car_detail.html", context={"car": car})
```

Comment:
Use class based view

Reply:
Can u explain how can i do this please. Some tips that can help me understand this